apps/fortuna/config.sample.yaml

```diff
@@ -12,6 +12,13 @@ chains:
     # How much to charge in fees
     fee: 1500000000000000
 
+    # Configuration for dynamic fees under high gas prices. The keeper will set
+    # on-chain fees to make between [min_profit_pct, max_profit_pct] of the max callback
+    # cost in profit per transaction.
+    min_profit_pct: 0
+    target_profit_pct: 20
```

Comment:
I'm not sure if we want this level of customisation. We can set this as the average of min_profit_pct and max_profit_pct.